dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -1114,10 +1114,7 @@ export const LiveLayout = ({
 								>
 									<RightColumn>
 										{renderAds && (
-											<Island
-												clientOnly={true}
-												deferUntil="visible"
-											>
+											<Island>
```

Comment:
Why is `deferUntil="visible"` gone?

Reply:
Good point. There's no need not to wait until the ad is in view